src/main/java/mate/academy/spring/config/AppConfig.java

```diff
@@ -49,7 +51,7 @@ public LocalSessionFactoryBean getSessionFactory() {
         props.put("hibernate.hbm2ddl.auto", env.getProperty("hibernate.hbm2ddl.auto"));
 
         factoryBean.setHibernateProperties(props);
-        factoryBean.setAnnotatedClasses(User.class, Book.class);
+        factoryBean.setAnnotatedClasses(User.class, Book.class, Author.class, Rent.class);
```

Comment:
we can use this line here:
factoryBean.setPackagesToScan("library.spring.entity");